### DISCUSSION TITLE
[MAUI] Newbie help

### DISCUSSION THREAD

**Discussion by leijae** [roubachof/Sharpnado.Tabs#108] [General]
Please forgive my newbiness here, longtime .NET MVC dev, but brand new to MAUI, and mobile app dev as a whole...

I'm misunderstanding a key part, I suppose. Can I use this in AppShell, or is this something separate? When I tried using it in Shell, I was getting an incompatibility error with IList.

**Comment by FreakyAli** [comment]
@leijae You can use it with Shell, there should be no compatibility issues whatsoever

**Reply by Mike-E-angelo** [reply]
I see that you can use it in a shell, but can you replace the shell tabs with these tabs?  That's really what I would like to do.  I looked in the demos and there is a `MauiShellSample` but it doesn't use these for the shell tabs. 🤔
